src/runtime/plugins/polyfill.client.ts

```diff
@@ -0,0 +1,16 @@
+import { defineNuxtPlugin } from "#imports";
+
+export default defineNuxtPlugin(async () => {
+  // shadowRootMode is the standardized attribute.
+  // but to support current Chrome, we need to check for shadowRoot as well.
+  // TODO: we can remove shadowRoot when Chrome 111 is released or is at least the last 2 released versions.
```

Comment:
I think we'll need to leave this feature detection in, rather than even plan to remove it. Users are on older browsers for a while and if someone is stuck on Chromium 90-111 then they'd just get a blank screen
```suggestion
  // shadowRootMode is the standardized attribute.
  // Chromium 90-111 supported shadowRoot, so we check for that as well.
```